src/components/ActionButtonGroup/index.ts

```diff
@@ -0,0 +1,2 @@
+export { default } from './ActionButtonGroup'
+export * from './ActionButtonGroup';
```

Comment:
I might be wrong, but I don't think you need to export anything other than the default here